access_policy.go

```diff
@@ -30,6 +30,7 @@ type AccessPolicy struct {
 
 	PurposeJustificationRequired *bool                 `json:"purpose_justification_required,omitempty"`
 	PurposeJustificationPrompt   *string               `json:"purpose_justification_prompt,omitempty"`
+	ApprovalRequired             *bool                 `json:"approval_required"`
```

Comment:
do you want to omitempty here too?

```suggestion
	ApprovalRequired             *bool                 `json:"approval_required,omitempty"`
```